doc/src/arch/reference.rst

```diff
@@ -2401,6 +2401,12 @@ The full format is documented below.
 
         .. note:: See ``from_switchpoint_order`` for value descritpions.
 
+    :opt_param switch_override:
+
+        Specifies the name of a switch to be used to override the wire_switch of the segments in the ``to`` set. Can be used to create more complicated wire structures such as T and L shaped segments.
```

Comment:
I'd change the "Can be used" sentence to: "Can be used to create switch patterns where different switches are used for different types of connections; by using a zero-delay and zero-resistance switch one can also create T and L shaped wire segments."